redirects.yaml

```diff
@@ -1429,17 +1089,6 @@ system/files/metal_sales_v3\.pdf/?: "https://insights.ubuntu.com/"
 system/files/u1/AutomatedDeploymentsWP-20090126\.pdf/?: "https://insights.ubuntu.com/"
 system/files/u1/server_report_v2\.pdf/?: "https://insights.ubuntu.com/"
 system/files/u35/Openbravo-final\.pdf/?: "https://insights.ubuntu.com/"
-tablet/app-ecosystem/?: "/mobile/developers"
-tablet/backed-by-canonical/?: "/partners/backed-by-canonical"
-tablet/operat/?: "/mobile"
-tablet/operators-and-oems/?: "/mobile/partners"
-tablet/register/?: "https://docs.google.com/a/canonical.com/forms/d/1VKTn1u2CifkTk7lQQC0Ws4tC7yaCep4DjRp34S6cV14/viewform"
-tablet/?: "/mobile"
-tablet/features/?: "/mobile/features"
-tablet/devices/?: "/mobile/devices"
-tablet/developers/?: "/mobile/developers"
-tablet/partners/?: "/mobile/partners"
-tablet/contact-us/?: "/mobile/contact-us"
```

Comment:
Shouldn't phone and tablet urls be added to the delected.yaml?

Reply:
I've added them as redirects to /mobile which is already in the deleted.yaml, feels like less code?

Reply:
Fair enough, thanks